RecoHGCal/TICL/plugins/PFTICLProducer.cc

```diff
@@ -52,6 +62,13 @@ void PFTICLProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptio
   desc.add<edm::InputTag>("trackTimeQualityMap", edm::InputTag("mtdTrackQualityMVA:mtdQualMVA"));
   desc.add<double>("timingQualityThreshold", 0.5);
   desc.add<bool>("useTimingAverage", false);
+  // For PFMuonAlgo
+  desc.add<edm::InputTag>("muonSrc", edm::InputTag("muons1stStep"));
+  desc.add<bool>("postMuonCleaning", false);
```

Comment:
The parameter `postMuonCleaning` is added here, but the corresponding `C++` variable is declared and initialized to `false`, w/o reading this parameter. Is that intended?

Reply:
Sorry for being slow responding.
Right. This is intended. We want to use PFMuonAlgo with postMuonCleaning =true only through PFProducer/PFAlgo.
But, in the upcoming update of this PR, I plan to update PFMuonAlgo so that we don't have to specify this manually any longer.